mmv1/products/securesourcemanager/Instance.yaml

```diff
@@ -85,6 +85,32 @@ examples:
       instance_id: 'my-instance'
       ca_pool_id: 'ca-pool'
       root_ca_id: 'root-ca'
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'secure_source_manager_instance_private_psc'
```

Comment:
Let's split the examples. One for connecting using endpoint; and one for connecting using NEG and ILB.

Reply:
Done, if you can double check